next/src/global/Seo.ts

```diff
@@ -1,2 +1,74 @@
-export const domain = "https://www.slowa-klucze.pl";
-export const locale = "pl_PL";
\ No newline at end of file
+import { type global } from '@/types/_pages/global';
+import { sanityFetch } from '@/utils/sanity-client';
+
+export const domain = 'https://www.slowa-klucze.pl';
+export const locale = 'pl_PL';
+
+const SEO = async ({
+  title,
+  description,
+  ogImage,
+  url,
+  ...props
+}: {
+  title: string;
+  description: string;
+  ogImage?: string;
+  url: string;
+}) => {
+  const {
+    seo: { og_Img },
+  } = await getData();
+
+  const seo = {
+    title: title || 'Słowa Klucze', // Zapytaj się Anety po projekcie
```

Comment:
Please write each comment in codebase with English

Reply:
I don't think this is even close to a mistake. Especially that this is a temporary comment and I'm the only person contributing to this project